markdown/3.x/en/faq.md

```diff
@@ -1,9 +1,32 @@
 ## General
 
+#### Question:
+Which version is the documentation referring to?
+#### Answer:
+Latest.
+
+---
 #### Question:
 I want to use FACT-Finder Web Components with Angular, but data binding used in the components conflicts with Angular's `{{ }}` binding. Can you fix it?
 #### Answer:
-Default binding delimiters can be changed via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine).
+You can add Angular's `ngNonBindable` attribute to the parent element to prevent Angular from parsing double curly braces. If you have to keep both Angular and FACT-Finder WebComponents bindings within the same element, you can use: 
+```html
+<ff-record> 
+    {{Angular Binding}}
+    <ng-container ngNonBindable>
+        {{FF Binding}}
+    </ng-container>
+</ff-record>
+```
+The `<ng-container>` tag will not be rendered in the DOM.
+
+Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that binding HTML still requires a single curly bracket, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
```

Comment:
When quickreading, I tripped over _"... binding HTML requires a single curly ..."_. Sounded a bit to me as if binding HTML would overwrite other ways of data binding. I find _"additional"_ helps understanding the workings even when you are not fully aware of the context.
Regarding _"single curly"_ vs _"curly"_, I'm rather indifferent. Both are fine.
Otherwise, looks good to me.
```suggestion
Alternatively, you can change the default FACT-Finder Web Components binding delimiters via `ff-communication`'s `mustache-delimiters` attribute. For more information please check the `Underlying Engine (Mustache)` section of the [Template Engine documentation](/documentation/3.x/template-engine). Please note that binding HTML still requires an additional pair of (single) curly brackets, what will cause an Angular error. You can utilize `ngNonBindable` to handle this issue as well:
```